src/Components/Home.js

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchAirlines } from '../redux/airlines/airlines';
+import Airline from './Airline';
+
+const Home = () => {
+  const dataList = useSelector((state) => state.airlines.airlines.data);
+  const dispach = useDispatch();
+
+  useEffect(() => {
+    dispach(fetchAirlines());
+  }, [dispach]);
+
+  return (
+    <section className="main">
+      <div className="header">
+        <h2>Flights list:</h2>
+        <p>Please book a flight from the following list</p>
+      </div>
+      <div className="container">
+        <div className="flights">
+          {!(dataList) ? <p>There are no flight yet!!</p>
```

Comment:
[OPTIONAL]
Is thereto delay in loading this until dataList for sure does not have data? I see there are no flights yet for a second or two before flights are loaded.